cmd/daemon/cert_refresh.go

```diff
@@ -0,0 +1,150 @@
+package daemon
```

Comment:
Would it be possible to add some tests for tis file? In case you wish to re-use it in Ory Hydra, it might also make sense to move this into ory/x so that it can be used across different repos :)